### PR TITLE
Feature footer

### PR DESCRIPTION
**Pull request by VoronaAV23**

### CODE REVIEW

File: css/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    list-style: none;
+    text-transform: uppercase;
+    border-left: 2px solid #0b2239;
+    padding-left: 15px;
+    margin-top: 25px;
+    letter-spacing: 0.2rem;
+
+}
+#footer-links ul li{
+        padding-bottom: 5px;
```
Comment (leonalkalai) [inline css/styles/style.css:40]: please remove extra indentation

File: css/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+#footer-links ul{
+    list-style: none;
+    text-transform: uppercase;
+    border-left: 2px solid #0b2239;
+    padding-left: 15px;
+    margin-top: 25px;
+    letter-spacing: 0.2rem;
+
```
Comment (leonalkalai) [inline css/styles/style.css:37]: please remove the extra empty line

File: css/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+#footer-links ul li{
+        padding-bottom: 5px;
+        font-weight: 700;
+}
+#footer-links ul li:last-child {
```
Comment (leonalkalai) [inline css/styles/style.css:43]: please keep one empty line between each element css declaration
*example*

element1{

}
....empty line....

element2{

}

Comment (VoronaAV23) [reply css/styles/style.css:53]: ok

Comment (VoronaAV23) [reply css/styles/style.css:62]: done

File: css/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    box-shadow: 0 0 12px 2px #0b2239;
+}
+
+.footer-box a{
+    color: #0b2239;
+    font-size: 15px;
+    list-style: none;
+    text-transform: uppercase;  
+    font-weight: 700;
+    \font-weight: 700;
```
Comment (MaryRichelle) [inline css/styles/style.css:105]: hey @VoronaAV23 this line seems  extra blocking our code  can we delete it please

**Review by MaryRichelle — APPROVED** [review]
i notice some responsiveness issue on small screens can we please fix this before we merge it or let us know if its a bug or feature?
footer-issue

File: css/styles/style.css (hunk 4 above)
Comment (VoronaAV23) [reply css/styles/style.css:105]: will fix it now

**Pull request merged**